cypress/fixtures/breakpoints.ts

```diff
@@ -7,10 +7,10 @@ type BreakpointSizes = {
 };
 
 const breakpointsToTest: Array<keyof typeof breakpoints> = [
-	'mobile',
+	// 'mobile',
 	'tablet',
-	'desktop',
-	'wide',
+	// 'desktop',
+	// 'wide',
```

Comment:
Do we want to reinstate these breakpoints?

Reply:
Good catch!